File: Algorithms/p389_Find_the_Difference.py
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        cnt = [0]*265
        for c in s:
            cnt[ord(c)] += 1
        for c in t:
            if cnt[ord(c)]==0:
                return c
            cnt[ord(c)] -= 1
        return chr(0)

File: Algorithms/p389_Find_the_Difference/p389_Find_the_Difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        vector<int> cnt(256);
        for(char c : s){
            cnt[c]++;
        }
        for(char c : t){
            if(cnt[c]==0){
                return c;
            }
            cnt[c]--;
        }
        return static_cast<char>(0);
    }
};
